src/rippled/lib/txSummary/NFTokenMint.test.js

```diff
@@ -1,631 +0,0 @@
-import NFTokenMint from './NFTokenMint'
```

Comment:
This should be moved to Transaction/NFTokenMint.  We don't want to loose these test cases.

Reply:
There were 2 tests existing and then 1 was added in the NFTokenMint parser PR, so I combined all 3 into one file (so nothing was lost)

Reply:
I think the difference is those were testing the parser directly.  Let me take a look at those

Reply:
Why the same assertions are happening the explanation of what is checked is lost.  CAn you add back in the context around those tests of the parsers?